src/lib/web3-utils.js

```diff
@@ -1,17 +1,18 @@
 import env from '../environment'
-import { providers as Providers } from 'ethers'
-import { id as keccak256 } from 'ethers/utils'
+import { providers as Providers, utils } from 'ethers'
 import { InvalidURI, InvalidNetworkType, NoConnection } from '../errors'
 import { validHttpFormat } from './uri-utils'
 
+const { id: hash256, solidityKeccak256: soliditySha3, toUtf8String } = utils
```

Comment:
Oh interesting, what is `hash256` if it's not `keccak256`? (Function signatures should be using keccak256)

Reply:
Ah you are right https://docs.ethers.io/v5/api/utils/hashing/#utils-id

Bad naming 😄